cpp/daal/src/algorithms/dtrees/dtrees_train_data_helper.i

```diff
@@ -119,7 +119,8 @@ struct SplitData
           nLeft(0),
           iStart(0),
           totalWeights(0.0),
-          leftWeights(0.0)
+          leftWeights(0.0),
+          left {}
```

Comment:
Isn't it best practice to instantiate in the same order we declare in the class?